apps/console/src/extensions/configs/models/common.ts

```diff
@@ -32,7 +32,8 @@ export interface CommonConfig {
          * Get the extensions for the header.
          * @returns Header extensions.
          */
-        getHeaderExtensions: () => Promise<HeaderExtension[]>;
+        getHeaderExtensions: (tenantDomain: string,
+            associatedTenants: any[]) => Promise<HeaderExtension[]>;
```

Comment:
Can we use a specific type other than any here?

Reply:
It's not possible ATM as type can be dynamic.